unpack-trees.c

```diff
@@ -1660,6 +1663,8 @@ int unpack_trees(unsigned len, struct tree_desc *t, struct unpack_trees_options
 done:
 	trace_performance_leave("unpack_trees");
 	clear_exclude_list(&el);
+	trace2_data_intmax("exp", NULL, "unpack_trees/nr_unpack_entries",
```

Comment:
It seems that sometimes, the experiments are prefixed with `"exp"`, but sometimes not. Did you mean to prefix all of them in that manner, or was it intentional to do that only with some, but not others?

Reply:
yeah, i've been inconsistent.  i was using "exp" for things i consider really temporary while we gather some data for a release or 2.  most of the other ones that have real categories, but have "experiment" in the commit message are things that i could keep and maybe upstream or could just drop.  i'm trying to not flood the telemetry stream with things that may not be useful long term, but at the same time gather enough data for our immediate analysis needs.